resources/shaders/basic.frag

```diff
@@ -0,0 +1,9 @@
+#version 120
+uniform vec4 color;
```

Comment:
I'm going to be annoying, but I like my whitespace lines. So after `#version` and before `void main()` as it gives a clear visual clue what things belong together.